src/coreComponents/mesh/SurfaceElementRegion.cpp

```diff
@@ -220,7 +220,8 @@ localIndex SurfaceElementRegion::addToFractureMesh( real64 const time_np1,
   for( auto const & setIter : faceManager->sets().wrappers() )
   {
     SortedArrayView< localIndex const > const & faceSet = faceManager->sets().getReference< SortedArray< localIndex > >( setIter.first );
-    SortedArray< localIndex > & faceElementSet = subRegion.sets().registerWrapper< SortedArray< localIndex > >( setIter.first ).reference();
+    SortedArray< localIndex > & faceElementSet = subRegion.sets().getReference< SortedArray< localIndex > >( setIter.first );
+    // faceElementSet.clear();
```

Comment:
@rrsettgast For all the set building I initially cleared the set because I mistakenly assumed the old behavior of `registerWrapper` would replace the previously held data (effectively calling `clear`). This worked fine everywhere else, in fact I still kept that behavior. But here this presented a problem with some integrated tests having diffs in the the size of `FractureElementSubregion/sets/externalSet`, but only in parallel.

My question is: do we actually want to clear the sets every time?

Reply:
Why it was originally a registration I don't know.

Alright, I'll remove the comment here and calls to `clear` elsewhere. My view though is that in these calls we are basically rebuilding the set from scratch, for consistencies sake it makes more sense to me that we first empty the set. Otherwise, as as evidenced by the error in `FractureElementSubregion/sets/externalSet` there is a disagreement in what is external between `FaceManager` and `FractureElementSubregion`.